app/views/users/index.html.haml

```diff
@@ -0,0 +1,32 @@
+.container  
+  %h1 Users
+  %p#notice= notice
+
+  %table.table.table-striped.table-bordered
+    %thead
+      %tr
+        %th Full Name
+        %th Email
+        %th Gender
+        %th Status
+        %th Last Shaved On
+        %th{colspan: 6} Actions
+
+
+    %tbody
+      - @users.each do |user|
+        %tr
+          %td= user.full_name
+          %td= user.email
+          %td= user.gender.capitalize
+          %td= user.status.try(:gsub, "_", " ").try(:capitalize)
```

Comment:
Try to stay away from any `logic` inside views. Use helpers for presentation logic. 
You chain two calls of `try`. This is a bad practice. Your model has validation which checks that `status` is present.